framework/resources/Functional/filter/selectivity/plan/plan.json

```diff
@@ -0,0 +1,20 @@
+{
+    "testId": "filter_selectivity_plan",
+    "type": "group",
+    "description": "Filter selectivity tests",
+    "categories": [ "smoke" ],
+    "matrices": [
+        {
+            "query-file": ".*.sql",
+            "schema": "dfs.drillTestDirTpch01Parquet",
+            "output-format": "tsv",
+            "expected-file": ".*.res",
+            "verification-type": [
+                "regex-compare"
```

Comment:
Can we call this "filter-ratio" or something on those lines. This for now is specific to the filter selectivity usecase.